### ISSUE TITLE
Help windows do not inherit NormalFloat highlight

### ISSUE THREAD

**Issue by akinsho** [nvim-orgmode/orgmode#510]
The highlights for the help menu don't match the same highlights used for the agenda/capture (if those are set to floats)

<img width="1329" alt="Screenshot 2023-02-25 at 09 16 44" src="https://user-images.githubusercontent.com/22454918/221349177-bff2bec8-828e-43bb-9486-522456229f30.png">

Other than the `win_split_mode = 'float'` and the border I haven't changed anything window related so I think the highlights just have different values set 🤔 

_Originally posted by @akinsho in https://github.com/nvim-orgmode/orgmode/issues/509#issuecomment-1445038705_

**Comment by kristijanhusak** [comment]
You are right, in some places we do `winhl` as `Normal:Normal`, while in other places we don't. I'm not sure which one of these options would be more preferred, I'm open for suggestions:

* Remove all `winhl`, use what user is generally using for `NormalFloat`
* Set `Normal:Normal` on all popup windows

**Comment by akinsho** [comment]
@kristijanhusak a pattern I've seen a lot of and is the best solution IMO is to link highlights to some custom highlight of yours that is advertised in the docs and linked as default e.g. `Normal:OrgmodeNormal` this allows the user to easily override the win highlights

**Comment by akinsho** [comment]
A smaller simpler solution is just to default to float normal, but the user then has no way to set their preferences
